pages/common/qmmp.md

```diff
@@ -0,0 +1,32 @@
+# qmmp
+
+> An audio player.
+> More information: <http://qmmp.ylsoftware.com>.
+
+- Launch qmmp:
+
+`qmmp`
+
+- Start/stop the currently playing audio:
+
+`qmmp --play-pause`
+
+- Seek forwards/backward a specific amount of time (in seconds):
+
+`qmmp --seek-{{fwd|bwd}} {{time_in_seconds}}`
+
+- Play the next audio file:
+
+`qmmp --next`
+
+- Play the previous audio file:
+
+`qmmp --previous`
+
+- Print the current volume:
+
+`qmmp --volume-status`
+
+- Increase/decrease the volume of the currently playing audio by 5 steps:
```

Comment:
```suggestion
- Increase or decrease the volume of the currently playing audio by 5 steps:
```


- This change is optional - no corresponding style guide requirement exists.
- Maybe it's better to use `or` instead of slash. Some maintainers in the past suggested me to do so.